runtime/vm/jvminit.c

```diff
@@ -551,6 +555,10 @@ exitJavaVM(J9VMThread * vmThread, IDATA rc)
 			TRIGGER_J9HOOK_VM_SHUTTING_DOWN(vm->hookInterface, vmThread, rc);
 		}
 
+#ifdef J9ZTPF
+		/* run z/TPF OS exit hook */
+		cjvm_jvm_shutdown_hook();
```

Comment:
Should this be done after runExitStages()?

Reply:
Similar to the previous comment.

I notice later on there is the same vm->exhitHook check and call.

Maybe it makes sense to put the call in the same location.